.github/CODEOWNERS

```diff
@@ -1,39 +1,39 @@
 * @DataDog/ruby-guild
 
 # Public Documentation
-/docs/Compatibility.md @DataDog/documentation
-/docs/GettingStarted.md @DataDog/documentation
+./docs/Compatibility.md @DataDog/documentation
```

Comment:
This will be a gotcha for folks editing this file in the future, given that adding the `./` prefix is not normal practice when writing the CODEOWNERS file.

I suggest adding a comment to this file explaining this to new folks adding entries here.